tests/test_cli_health_check_coverage.py

```
@@ -0,0 +1,223 @@
+    def test_probe_timeout_float_returns_true(self):
+        config = _base_config(health_check_value={"probe_timeout": 5.5})
+        self.assertTrue(self._run_check_config(config))
+
+    def test_probe_timeout_small_positive_returns_true(self):
+        config = _base_config(health_check_value={"probe_timeout": 0.001})
+        self.assertTrue(self._run_check_config(config))
+
+    def test_error_message_health_check_not_dict(self):
+        config = _base_config(health_check_value="not_a_dict")
+        with patch("builtins.print") as mock_print:
+            with patch("mmrelay.cli._validate_credentials_json", return_value=False):
```

Ruff (0.15.9)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)